### ISSUE TITLE
Why is Git Repository URL set to readonly?

### ISSUE THREAD

**Issue by davidchristiandy** [bitrise-io/steps-git-clone#79]
I have a requirement to clone from another repository.

**Comment by viktorbenei** [comment]
Hi @davidchristiandy ,

Please don’t use multiple **Git Clone** steps to clone multiple repos!

**Use Git Clone only for the primary repository**, and either add the other repos as **git submodules** to the primary repository, or use a **Script** step to `git clone` the other repos.

This is because Git Clone is configured to checkout the branch, commit etc. the build is started with, and if you'd add a second Git Clone step to the workflow that would try to checkout the same commit/tag/.. the build was started with for the primary repository.

So, in short, please use only one Git Clone step in your workflow, to "clone" the primary repository, and either add the other repos as submodules to the primary, or use a Script step to clone the other repos.

If you'd have any questions just let us know, or feel free to share your use case, why you'd need the second repo, e.g. which branch of it would you need or what params would be used and we'll try suggest the best solution for the use case :)
